.gitattributes

```diff
@@ -28,5 +28,7 @@ configure				-whitespace
 src/backend/distributed/utils/citus_outfuncs.c -citus-style
 src/backend/distributed/utils/citus_read.c -citus-style
 src/backend/distributed/utils/citus_readfuncs_95.c -citus-style
+src/backend/distributed/utils/citus_readfuncs_96.c -citus-style
```

Comment:
Gna. I think it's a bad idea to it by first adding and then removing stuff, didn't we actually discuss this?